hazelcast/src/main/java/com/hazelcast/instance/Node.java

```diff
@@ -799,4 +801,12 @@ public BuildInfo getBuildInfo() {
         return attributes;
     }
 
+    private void logGroupPasswordInfo() {
+        if (!isNullOrEmpty(config.getGroupConfig().getPassword())) {
```

Comment:
Does it make sense to define a default group password when we encourage users to not define any? Maybe `GroupConfig.DEFAULT_GROUP_PASSWORD` should be `""`?

Reply:
IMO we should keep it for backward compatibility reasons. It's also used in EE for default JAAS authentication when security is enabled.